react/AppSections/components/AppTile.styl

```diff
@@ -0,0 +1,76 @@
+.AppTile-icon-wrapper
+  margin-top .25rem
+  margin-bottom .25rem
+  width 3rem
+  height 3rem
+
+.AppTile-icon-wrapper .AppTile-icon
+  object-fit contain
+  max-width 100%
+  width auto
+
+.AppTile-icon--default
+  height 100%
+  padding 0 .5em
+
+.AppTile-desc
+  display flex
+  flex-direction column
+  justify-content flex-start
+  align-items center
+  margin-top .5rem
+  text-align center
+
+.AppTile-title, .AppTile-developer, .AppTile-status
+  display block
+  margin 0
+  text-overflow ellipsis
+  white-space nowrap
+  overflow hidden
+
+.AppTile-title
+  height 1.25rem
+  color var(--black)
+
+.AppTile-developer
+  height 1rem
+  font-size 12px
```

Comment:
`px`? Shouldn't we use `rem` instead? Also, we have a few stylus variables for font-size, we should use them (https://github.com/cozy/cozy-ui/blob/master/stylus/settings/typography.styl#L14)

Reply:
Ah yes true. It's not code that has been written in this PR, just code that has been moved, this is why I did not refactor much more that the split between Tile and AppTile.

Anyway, thanks for the suggestions : https://github.com/cozy/cozy-ui/pull/1521